VERSION

```diff
@@ -1 +1 @@
-3.1.0
+3.2.0
```

Comment:
Would it make sense to do the release procedure in a separate PR? I think that's how we've done in the past I don't remember if pushing version bump alongside code changes has any downside, so no strong opinion, I guess you can try either way.

Reply:
my general philosophy is to have each PR be self contained and releasable, even if we decide not to actually release it by itself.  For a couple reasons, 1) if every PR is releasable, then it isn't blocking anybody's changes that come after, and 2) so we don't forget to do something by saving it for later.